lib/src/widgets/setting_actions.dart

```diff
@@ -29,6 +29,7 @@ class SettingActions {
     connectionSettingAction,
     walletSettingAction,
     addressBookSettingAction,
+    silentPaymentsSettingAction,
```

Comment:
need to also add the check for whether this wallet supports Silent Payments or not in the page that uses this list